benchmarking/benchmark.py

```diff
@@ -54,49 +54,101 @@ def get_times_skfp(X: np.ndarray, transformer_cls: type, **kwargs) -> np.ndarray
     return np.array(result).reshape((len(N_CORES), N_SPLITS))
 
 
-def save_results(
+LEGAL_PLOT_TYPES = ["time", "speedup", "fprints_per_second"]
```

Comment:
consider ALLOWED_PLOT_TYPES or just PLOT_TYPES

Reply:
+1

Reply:
Also, maybe `fps_per_second`? We use `fps` variable quite a bit